static/beta/itless/navigation/rhel-navigation.json

```diff
@@ -161,6 +161,14 @@
               "href": "/insights/malware/systems",
               "product": "Red Hat Insights"
             }
+          ],
+          "alt_title": [
```

Comment:
This entry is never used directly in the `services.json` files this will never get uploaded to the search index. Either find a different link which is referenced or create a new search entry in the `static-services-entries.json` file.

Reply:
Let me know if the changes that I've made is fine!